tests/Unit/Evolution/Systems/GeneralizedHarmonic/Test_Tags.cpp

```diff
@@ -0,0 +1,62 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "tests/Unit/TestingFramework.hpp"
+
+#include <string>
+
+#include "Evolution/Systems/GeneralizedHarmonic/Tags.hpp"
+
+namespace {
+struct ArbitraryFrame;
+}  // namespace
+
+#define _TEST_SIMPLE_TAGS(D, F)                                            \
+  CHECK(GeneralizedHarmonic::Tags::Pi<D, F>::name() == "Pi");              \
```

Comment:
We have been moving away from calling `name` directly and use `db::tag_name` instead.

Reply:
changed